quiz/main.go

```diff
@@ -0,0 +1,81 @@
+package main
+
+import (
+	"bufio"
+	"encoding/csv"
+	"flag"
+	"fmt"
+	"log"
+	"math/rand"
+	"os"
+	"strings"
+	"time"
+)
+
+type Quiz struct {
+	filename       string
+	timeLimit      int
+	shuffle        bool
+	problems       [][]string
+	totalScore     int
+	totalQuestions int
+}
+
+func (q *Quiz) readProblems() {
+	csvfile, err := os.Open(q.filename)
```

Comment:
You should probably make sure you close the file after opening it.
`defer csvfile.Close()`